packages/react-component-library/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -297,11 +297,11 @@ describe('DatePicker', () => {
             expect(onChange).toHaveBeenCalledTimes(2)
             expect(onChange).toHaveBeenCalledWith({
               startDate: new Date('2019-12-01T00:00:00.000Z'),
-              endDate: undefined,
+              endDate: new Date('2020-01-20T12:00:00.000Z'),
```

Comment:
This seems wrong – needs further investigation...

Reply:
This is caused by https://github.com/Royal-Navy/design-system/issues/1391, the test is actually changing the end date twice instead of changing the start date and then the end date.